spec/cache/asset_spec.rb

```diff
@@ -59,4 +59,12 @@
       expect(new_cache.asset_for_node(node)).to eq(asset_name)
     end
   end
+
+  describe '#unassign_asset' do
+    it 'unassigns the asset from the node' do
+      cache.unassign_asset('asset_test')
+      new_cache = Metalware::Cache::Asset.new
+      expect(new_cache.data).not_to eq(content)
+    end
```

Comment:
Can we also add a spec for what happens if the asset is missing? Mainly ensure that it doesn't error